docs/docs/configuration/alpha_config.md

```diff
@@ -414,6 +414,7 @@ Provider holds all configuration for a single provider
 | `provider` | _[ProviderType](#providertype)_ | Type is the OAuth provider<br/>must be set from the supported providers group,<br/>otherwise 'Google' is set as default |
 | `name` | _string_ | Name is the providers display name<br/>if set, it will be shown to the users in the login page. |
 | `caFiles` | _[]string_ | CAFiles is a list of paths to CA certificates that should be used when connecting to the provider.<br/>If not specified, the default Go trust sources are used instead |
+| `caFilesAppend` | _bool_ | CAFilesAppend determines if CAFiles are appended to the default store<br/>If set to true, CAFiles will be appended to the default Go trust stores, otherwise it will replace it |
```

Comment:
```suggestion
| `useSystemTrustStore` | _bool_ | UseSystemTrustStore determines if your custom CA files and the system trust store are used<br/>If set to true, your custom CA files and the system trust store are used otherwise only your custom CA files. |
```